plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/CallTemplateMediatorDeserializer.java

```diff
@@ -46,7 +46,7 @@ public CallTemplateMediator createNode(IGraphicalEditPart part, AbstractMediator
         setElementToEdit(mediatorModel);
         setCommonProperties(mediator, mediatorModel);
         executeSetValueCommand(CALL_TEMPLATE_MEDIATOR__TARGET_TEMPLATE, mediator.getTargetTemplate());
-        executeSetValueCommand(CALL_TEMPLATE_MEDIATOR__AVAILABLE_TEMPLATES, "Select From Templates");
+        executeSetValueCommand(CALL_TEMPLATE_MEDIATOR__AVAILABLE_TEMPLATES, "");
```

Comment:
Is there a specific reason to remove the message?

Reply:
This is not a message. This was used as a default value for the CALL_TEMPLATE_MEDIATOR__AVAILABLE_TEMPLATES property. We have removed insensible default value properties with this PR.